tools/test_tpm_ek_chain.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+WORKDIR="$(mktemp -d)"
+declare -i MAX_CERT_SIZE=0
+declare -i CHAIN_LEN=0
+declare -i CHAIN_SIZE=0
+
+function cleanup {
+	rm -rf $WORKDIR
+	exit
+}
+
+# tpm2-tools changed command and option names. This script was tested with
+# versions 5.2 and 3.0.2, and may not work on different versions. Contibutions
+# welcome.
+which tpm2_pcrread > /dev/null
+if [ $? -eq 0 ]; then
+	# 5.2
+	pcrread="tpm2_pcrread"
+	nvread="tpm2_nvread -C o"
+else
+	# 3.0.2
+	pcrread="tpm2_pcrlist"
+	nvread="tpm2_nvread -x"
+fi
+
+# Do we have working TPM2.0?
+$pcrread >/dev/null 2>&1
+if [ $? -ne 0 ]; then
+	echo "Command execution failed, is TPM2.0 available?"
```

Comment:
@krystian-hebel It fails this way wven when /dev/tpm0 is there, but tpm2-tools are not installed. We can improve logic and/or message

Reply:
Added test for tools availability.